app/src/main/res/xml/preferences_general.xml

```diff
@@ -77,6 +77,10 @@
             android:summary="@string/settings_debug_description"
             android:title="@string/settings_debug_label" />
 
+        <Preference
+            android:summary=""
+            android:title="" />
+
```

Comment:
Is the idea here to have this as "padding"?" Could we add padding to `settings_fragment.xml` instead?

Reply:
This is from the fake padding commit. I've used this because it is enough to prevent the Debug Settings from being obscured by the 3 button navigation bar when scrolling all the way down but leaves a bit more than enough space for the navigation pill. Attempting to place padding from my attempts caused the settings view to not render below the navigation bar, which was my main goal.

Here are some photos to show what I mean:

(screenshots from Pixel 7)
| Current commit with the "fake" padding applied | Padding in `settings_fragment.xml` |
| :---: | :---: |
| ![NavBarCurrent](https://github.com/d4rken-org/capod/assets/38306929/96ef7d0a-65f6-4779-add8-d2b60d39fae3) Content renders below the navigation pill | ![NavBarIssue](https://github.com/d4rken-org/capod/assets/38306929/28c0e2a7-0d23-45df-813b-abee7d78ec9e) The navigation pill becomes the background colour as the padding pushes the entire view upwards, thus ending the scroll view where the navigation pill starts |

By holding down on the Debug Settings to see its size, it is exactly aligned with the 3 button bar (screenshot from Pixel XL):
<img src="https://github.com/d4rken-org/capod/assets/38306929/1023fd4f-c552-4ab8-983f-96d2c2727950" width='250'/>

========================

Although intended to be a temporary solution, it worked quite well, so I left it in the commit.